<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUser</class>
 <widget class="QDialog" name="CreateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creación de usuario</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontal_layout_buttons">
    <item>
     <widget class="QPushButton" name="push_button_clean">
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_button_save">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>261</width>
     <height>192</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid_layout_form">
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_edit_paternal_surname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_paternal_surname">
      <property name="text">
       <string>Apellido Paterno:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_password">
      <property name="text">
       <string>Contraseña: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_edit_name"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_user_type">
      <property name="text">
       <string>Tipo de usuario:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>Nombre:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_maternal_surname">
      <property name="text">
       <string>Apellido Materno:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_repeat_password">
      <property name="text">
       <string>Repetir Contraseña:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="line_edit_password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="line_edit_maternal_surname"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="combo_box_user_type"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="line_edit_repeat_password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_user_name">
      <property name="text">
       <string>Nombre de usuario:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="line_edit_user_name"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
